src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -158,23 +162,52 @@ public BukkitEventValues() {}
 	private static final boolean NAMESPACE_SUPPORT = Skript.classExists("org.bukkit.NamespacedKey");
 	
 	static {
-		
+		// ChunkEvents
+		EventValues.registerEventValue(ChunkEvent.class, Chunk.class, new Getter<Chunk, ChunkEvent>() {
```

Comment:
Just curious,

How did you arrive at some of the priority values? Is there a reason for these ones to not use the default priority?

Reply:
Depending on their registering time and the reliance on other event values. Like an entity to player.